build.gradle

```diff
@@ -66,7 +66,7 @@ subprojects {
     }
 
     test {
-        reports.html.enabled = false
+        reports.html.enabled = true
```

Comment:
I think this wont cause problems with the overall build... I have a ticket open to make sure both sets of reports work equally well.